core/src/main/java/google/registry/tools/CreateOrUpdateBulkPricingPackageCommand.java

```diff
@@ -56,9 +56,11 @@ abstract class CreateOrUpdateBulkPricingPackageCommand extends MutatingCommand {
   @Nullable
   @Parameter(
       names = "--next_billing_date",
+      converter = DateTimeParameter.class,
+      validateWith = DateTimeParameter.class,
       description =
           "The next date that the bulk pricing package should be billed for its annual fee")
-  Date nextBillingDate;
+  DateTime nextBillingDate;
```

Comment:
This seems like a bad change to me?  Why are we going to make people specify an entire date time on the command line when the only thing that should be passed in is a date?